### PR TITLE
Add pipeline to replace execution from old zuul's crontab

### PR DESCRIPTION
**Pull request by rgarrigue**

### CODE REVIEW

**Comment by rgarrigue** [comment]
It works

https://jenkins.wazo.io/job/teambot/

image

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    MATTERMOST_HOOK_URL = credentials("mattermost-teambot-hook")
+  }
+
+  stages {
+
+    stage ('Prepare') {
+      steps {
+        script {
+          // Rename build
+          // currentBuild.displayName = ""
```
Comment (fblackburn1) [inline Jenkinsfile]:
```suggestion
```

**Review by fblackburn1 — CHANGES_REQUESTED** [review]

**Comment by fblackburn1** [comment]
Does schedule is in UTC? Because zuul was in UTC but jenkins-private is at the quebec timezone

**Comment by rgarrigue** [comment]
Good catch, it'll rely on Jenkins timezone. Can't seems to force that in the Jenkinsfile to be explicit. So updated the cron hour value in the PR to match current behavior

**Review by fblackburn1 — APPROVED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
```
Comment (fblackburn1) [inline Jenkinsfile]: you force push and re-add these lines ...

**Review by fblackburn1 — CHANGES_REQUESTED** [review]

**Review by fblackburn1 — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (fblackburn1) [reply Jenkinsfile]: fixed

**Pull request merged**